app/src/components/Button/primary/primary.test.tsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
-import Primary, {PrimaryProps} from './primary';
-import {render as testRender} from '@testing-library/react-native';
+import Primary, { PrimaryProps } from './primary';
+import { fireEvent, render as testRender, screen } from '@testing-library/react-native';
+import { Screen } from '@react-navigation/elements';
+
 
-const render = (props?: Partial<PrimaryProps>) =>
-  testRender(<Primary text="text" onPress={() => {}} {...props} />);
 
 it('renders correctly', () => {
```

Comment:
You've written a good test but have forgotten to rename it. Maybe 'run onPress when clicked' would've been better

Reply:
sure, I will do it next time.